<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderSettingsDialog</class>
 <widget class="QDialog" name="RenderSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>244</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Render settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Camera</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Pcam</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pcam_x">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pcam_y">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pcam_z">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Pview</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pview_x">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pview_y">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pview_z">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Vup</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="vup_x">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="vup_y">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="vup_z">
                    <property name="decimals">
                     <number>2</number>
                    </property>
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="stepType">
                     <enum>QAbstractSpinBox::DefaultStepType</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Fone</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>R</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="red">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>G</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="green">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>B</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="blue">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Viewport</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>zn</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="zn">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>-999.990000000000009</double>
                  </property>
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="stepType">
                   <enum>QAbstractSpinBox::DefaultStepType</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>zf</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="zf">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>-999.990000000000009</double>
                  </property>
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="stepType">
                   <enum>QAbstractSpinBox::DefaultStepType</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>sw</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="sw">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>-999.990000000000009</double>
                  </property>
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="stepType">
                   <enum>QAbstractSpinBox::DefaultStepType</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>sh</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="sh">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>-999.990000000000009</double>
                  </property>
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="stepType">
                   <enum>QAbstractSpinBox::DefaultStepType</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Quality</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="rough">
                <property name="text">
                 <string>Rough</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="normal">
                <property name="text">
                 <string>Normal</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="fine">
                <property name="text">
                 <string>Fine</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Other</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Gamma</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="gamma">
                  <property name="maximum">
                   <double>5.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Depth</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="depth">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
